layouts/Layout.module.scss

```
@@ -7,3 +7,62 @@
   overflow-y: auto;
   background: #f8f8f8;
 }
+
+
+// auth styles
+
+.imgContainer{
+  position: relative;
+  display: block;
+  width: 100%;
+  height: 500px;
+  margin: 0 auto;
+ 
+
+}
+.img{
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-size: cover;
+  background-position: center top;
+  background-repeat: no-repeat;
+  -webkit-animation:fadeImg 12s ease-in infinite ;
+  animation: fadeImg 12s ease-in infinite ;
+
+}
+
+#pic1 {
+  animation-delay: 0s;
+  background-image: url('../assets/africanwomansmiling.png');
+}
+#pic2{
+  animation-delay: 4s;
+  background-image: url('../assets/africanwoman.png');
+
+}
+
+#pic3 {
+  animation-delay: 8s;
+  background-image: url('../assets/man.png');
+
+}
+
+@keyframes fadeImg {
+  0% {
```

this fadeIn animation is incorrect.
```suggestion
 from {
  opacity:0;
  }
  to{
  opacity:100%;
  }
```

---

Thank you for pointing this out @copy-ninja1 .

I did try out your sugesstion though and it didnt seem to work, the transistion between the images isnt smooth and one can visbily see the images as they are stacked on each other as they disappear(opacity ).


i  think the below peice of code does work better .

`0%{
  opacity: 1;
 }
 20%{
  opacity: 0;
 }
 40%{
  opacity: 0;
 }
 60%{
  opacity: 0;
 }
 80%{
  opacity: 0;
 }
 100%{
  opacity: 1;
 }`